README.md

```diff
@@ -168,7 +169,7 @@ oc patch ds fluentd -p "spec:
       - name: fluentd
         securityContext:
           privileged: true"
-oc delete pod -l k8s-app=fluentd-logging
+oc delete pod -l k8s-app = fluentd-logging
```

Comment:
Spaces around equal causes oc delete pod command failure.
Cloud you delete them?